main.go

```diff
@@ -12,13 +12,16 @@ import (
 )
 
 var (
-	version    = "dev"
-	app        = kingpin.New("svu", "semantic version util")
-	nextCmd    = app.Command("next", "prints the next version based on the git log").Alias("n").Default()
-	majorCmd   = app.Command("major", "new major version")
-	minorCmd   = app.Command("minor", "new minor version").Alias("m")
-	patchCmd   = app.Command("patch", "new patch version").Alias("p")
-	currentCmd = app.Command("current", "prints current version").Alias("c")
+	version      = "dev"
+	app          = kingpin.New("svu", "semantic version util")
+	nextCmd      = app.Command("next", "prints the next version based on the git log").Alias("n").Default()
+	majorCmd     = app.Command("major", "new major version")
+	minorCmd     = app.Command("minor", "new minor version").Alias("m")
+	patchCmd     = app.Command("patch", "new patch version").Alias("p")
+	currentCmd   = app.Command("current", "prints current version").Alias("c")
+	noMetadata   = app.Flag("no-metadata", "discards pre-release and build metadata").Bool()
+	noPreRelease = app.Flag("no-pre-release", "discards pre-release metadata").Bool()
+	noBuild      = app.Flag("no-build", "discards build metadata").Bool()
```

Comment:
I think you can use something like

```suggestion
	noBuild      = app.Flag("build", "discards build metadata").Default(true).Bool()
```

then it will auto create both `--build` and `--no-build` flags

Reply:
Done!